### ISSUE TITLE
The library depends on old version of glob-parent with security vulnerabilities

### ISSUE THREAD

**Issue by rtau-cca** [Cody2333/koa-swagger-decorator#131]
With this library installed, `npm audit` shows that it depends on glob-parent which contains vulnerabilities:
```
$ npm audit

                       === npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular expression denial of service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ glob-parent                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=5.1.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ koa-swagger-decorator                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ koa-swagger-decorator > @types/globby > fast-glob >          │
│               │ glob-parent                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1751                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Regular expression denial of service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ glob-parent                                                  │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=5.1.2                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ koa-swagger-decorator                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ koa-swagger-decorator > globby > fast-glob > glob-parent     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1751                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

FYI, the glob-parent 3.1.0 is installed.

**Comment by rtau-cca** [comment]
The vulnerabile library is still used in v1.8.0

**Comment by Cody2333** [comment]
should fixed in v1.8.2
